src/components/ToDoItemEdit.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+  <div class="">
+    <div
+      class="relative sm:mx-auto mx-[16px] flex flex-col space-y-8 justify-center rounded-2xl bg-white border-black border-2 p-[18px] m-[50px] sm:w-[610px] w-[320px]"
+    >
+      <div class="flex w-full justify-between flex-row">
+        <div class="flex flex-col">
+          <input
+            class="w-full placeholder-black leading-[3rem] sm:text-5xl text-lg flex mt-4 font-bold"
+            type="text"
+            placeholder="Title"
+          />
+          <div class="flex w-full items-center sm:h-[12px] h-[5px] lg:hidden">
+            <img :src="dateIcon" class="lg:hidden mr-1 h-3 w-3" />
+            {{ newDate() }}
+          </div>
+        </div>
+
+        <div class="mr-[5px]">
+          <button
+            @click="setPriority('Low')"
+            class="flex justify-center bg-green-500 rounded-2xl w-[10px] h-[10px] sm:invisible visible"
+            :class="{
+              'border border-black': 'Low' == toDo.priority,
+            }"
+          ></button>
+        </div>
+        <div class="mr-[5px]">
+          <button
+            :class="{
+              'border border-black': 'Medium' == toDo.priority,
+            }"
+            @click="setPriority('Medium')"
+            class="flex justify-center bg-yellow-500 rounded-2xl w-[10px] h-[10px] sm:invisible visible"
```

Comment:
Please create a BaseButton component and add these classes as default to it. You writing the same classes multiple times